engine/src/flutter/shell/platform/windows/flutter_windows_view_unittests.cc

```diff
@@ -162,9 +162,12 @@ TEST(FlutterWindowsViewTest, SubMenuExpandedState) {
   root.decreased_value = "";
   root.child_count = 0;
   root.custom_accessibility_actions_count = 0;
-  root.flags = static_cast<FlutterSemanticsFlag>(
-      FlutterSemanticsFlag::kFlutterSemanticsFlagHasExpandedState |
-      FlutterSemanticsFlag::kFlutterSemanticsFlagIsExpanded);
+  auto flags = FlutterSemanticsFlags{
+      .has_expanded_state = true,
+      .is_expanded = true,
+  };
```

Comment:
My C++ is getting rusty already, but I believe our style is:

```suggestion
  FlutterSemanticsFlags flags = {
      .has_expanded_state = true,
      .is_expanded = true,
  };
```

Same comment for below

Reply:
Oh nope never mind, Chris posted above that this is the correct style. Definitely prefer Chris's comments over mine. Closing!